tests/fixtures/noqa/noqa.py

```diff
@@ -703,3 +703,8 @@ def consecutive_yields():
         # https://github.com/wemake-services/wemake-python-styleguide/issues/1082
         break
     my_print(4)
+
+
+class Memo0Output(object):  # noqa: WPS124
+    def __init__(self):
```

Comment:
Can be replaced with just `anti_wps4?? = 'unreadable class'` (sorry, I don't remember the violation number)